src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php

```diff
@@ -1 +1 @@
-<?php echo $view['form']->block($form, 'widget_attributes') ?>
+<?php echo $view['form']->block($form, 'widget_attributes');
```

Comment:
This one is perfectly valid and I won't change it. The same goes for all PHP templates below. I think there is a big difference between PHP classes (where we should avoid the ending `?>`) and PHP templates, where you mostly have a bunch of PHP code within HTML/text, ... In the case of PHP templates, having all PHP code properly wrapped is important. I know that this PHP template and some others in Symfony are a bit extreme and edge cases, but nonetheless, they are templates and should be treated as such.

There are many ways to make the difference, but I think the easiest one is when the file only contains one big chunk of PHP and nothing else (big chunk meaning that we are talking about a significant amount of PHP).

Thoughts?